drift/info_parser.py

```diff
@@ -131,47 +135,53 @@ def _get_value_name(name):
                 return group
 
-    # build out group names
-    with PT_GC_BUILD_GROUP_NAMES.time():
-        group_names = {_get_group_name(c["name"]) for c in comparisons if "." in c["name"]}
-        grouped_comparisons = []
-        for group_name in group_names:
-            grouped_comparisons.append({"name": group_name, "comparisons": []})
-
-    # populate groups
-    with PT_GC_POPULATE_GROUPS.time():
-        for comparison in comparisons:
-            if "." in comparison["name"]:
-                group = _find_group(_get_group_name(comparison["name"]))
-                comparison["name"] = _get_value_name(comparison["name"])
-                group["comparisons"].append(comparison)
-                group["comparisons"] = sorted(
-                    group["comparisons"], key=lambda comparison: comparison["name"]
-                )
+    def _set_summary_state(grouped_comparison):
+        states = {comparison["state"] for comparison in grouped_comparison["comparisons"]}
+        if COMPARISON_DIFFERENT in states:
+            grouped_comparison["state"] = COMPARISON_DIFFERENT
+        elif COMPARISON_INCOMPLETE_DATA in states:
+            if COMPARISON_SAME in states:
+                grouped_comparison["state"] = COMPARISON_DIFFERENT
             else:
-                grouped_comparisons.append(comparison)
-
-    # set summary state if grouped comparison contains groups
-    with PT_GC_SET_SUMMARY.time():
-        for grouped_comparison in grouped_comparisons:
-            if "comparisons" in grouped_comparison:
-                states = {comparison["state"] for comparison in grouped_comparison["comparisons"]}
-                if COMPARISON_DIFFERENT in states:
-                    grouped_comparison["state"] = COMPARISON_DIFFERENT
-                elif COMPARISON_INCOMPLETE_DATA in states:
-                    if COMPARISON_SAME in states:
-                        grouped_comparison["state"] = COMPARISON_DIFFERENT
-                    else:
-                        grouped_comparison["state"] = COMPARISON_INCOMPLETE_DATA
-                elif COMPARISON_SAME in states and len(states) == 1:
-                    grouped_comparison["state"] = COMPARISON_SAME
-                else:  # use 'incomplete data' as the fallback state if something goes wrong
-                    grouped_comparison["state"] = COMPARISON_INCOMPLETE_DATA
+                grouped_comparison["state"] = COMPARISON_INCOMPLETE_DATA
+        elif COMPARISON_SAME in states and len(states) == 1:
+            grouped_comparison["state"] = COMPARISON_SAME
+        else:  # use 'incomplete data' as the fallback state if something goes wrong
+            grouped_comparison["state"] = COMPARISON_INCOMPLETE_DATA
+
+    # with PT_GC_BUILD_GROUP_NAMES.time(): # move to proper place or remove competely
+    # with PT_GC_POPULATE_GROUPS.time(): # move to proper place or remove competely
+    # with PT_GC_SET_SUMMARY.time(): # move to proper place or remove competely
```

Comment:
@petrblaho  May we add a measurement here?

Reply:
then we would be measuring the same as this https://github.com/RedHatInsights/drift-backend/pull/325/files/d2d06c01cf43768358e8f5b84a1b30d1f2b85a50#diff-d94d1e0636fff7ab5a4d4f6f24ff091770a8a2ec2c170453ec644766a4f16857R86